File: atv-topico-3-4/topico-3/iguais.hs
iguais d e f = 
  if (d == e && e == f)
    then 3
    else if (d == e && e /= f)
      then 2
      else if (d == f && f /= e)
        then 2
        else 0
main = do
    a <- readLn :: IO Int
    b <- readLn :: IO Int
    c <- readLn :: IO Int
    print $ iguais a b c

File: atv-topico-3-4/topico-3-js/iguais.js
const iguais = (d, e, f) => {
  if (d == e && e == f) {
    return 3;
  } else if (d == e && e != f) {
    return 2;
  } else if (d == f && f != e) {
    return 2;
  } else {
    return 0;
  }
};

console.log(iguais(3, 2, 1));
